### PR TITLE
More PAI foods

### PR DESCRIPTION
**Pull request by RealestEstate**
Adds some more foods to the PAI food synth, as well as 4 different PAI shaped cookies that can only be made by the PAI.

PAI cookie sprites courtesy of Ronnie Pickering.

[content]

:cl:
 * rscadd: PAI food synths now have more food options

### CODE REVIEW

**Comment by PrimeDSS13** [comment]
mmmm cookie

**Comment by SonixApache** [comment]
jesus that's a lotta paicreep

**Comment by PrimeDSS13** [comment]
also yeah, what Sonix said. You might be adding *too* many foods to the pAI.

**Comment by RealestEstate** [comment]
yeah but they're mostly harmless and having several options in a roleplaying game is in my opinion a good thing

**Comment by SonixApache** [comment]
mints and peanuts kill people, dan stuff ain't too healthy
a lot of those items require actual interaction with certain things which is better for rp

giant :+1: on the cookies but trim some of the rest a bit maybe

**Comment by ghost** [comment]
>fortune cookies, chocolate eggs
>SYNTHMEAT

too powerful

**Comment by yclatious** [comment]
Slim it down just a little but tbh, the fucking borers can also emp a motherfucker, shoot rezadone up their ass and more depending on where the slug crept up so fuck it.

**Comment by RealestEstate** [comment]
Removed Synthmeat, Mints, Dan food.

**Comment by jknpj** [comment]
I can understand the cookies, but i cannot understand why pAIs need to become portable hydroponic trays.

File: code/modules/mob/living/silicon/pai/pai.dm [hunk 1]
```diff
@@ -53,8 +53,27 @@
...
+		"Chocolate Bar" = /obj/item/weapon/reagent_containers/food/snacks/chocolatebar,
+		"Space Twinkie" = /obj/item/weapon/reagent_containers/food/snacks/spacetwinkie,
+		"Sweets" = /obj/item/weapon/reagent_containers/food/snacks/sweet,
+		"Gum" = /obj/item/gum,
+		"Popcorn" = /obj/item/weapon/reagent_containers/food/snacks/popcorn,
+		"Faggot" = /obj/item/weapon/reagent_containers/food/snacks/faggot,
+		"Sausage" = /obj/item/weapon/reagent_containers/food/snacks/sausage,
+		"Buttered Toast" = /obj/item/weapon/reagent_containers/food/snacks/butteredtoast,
+		"Egg" = /obj/item/weapon/reagent_containers/food/snacks/egg,
+		"Chocolate Egg" = /obj/item/weapon/reagent_containers/food/snacks/chocolateegg,
```
Comment (jknpj) [inline code/modules/mob/living/silicon/pai/pai.dm]: >Egg AND choco egg

:thinking:

**Comment by RealestEstate** [comment]
>portable hydro trays

unfair considering they can synthesize bananas, but sure I’ll remove the chili and eggs.

File: code/modules/mob/living/silicon/pai/pai.dm (hunk 1 above)
Comment (RealestEstate) [reply code/modules/mob/living/silicon/pai/pai.dm]: the components to a choco egg are egg and chocolate so naturally you combine them when synthing

**Comment by RealestEstate** [comment]
removed chilis, eggs, chocolate egg, buttered toast. It dawned on me that one of the cookie sprites was actually meant to be a toast sprite I think so I made a separate pai toast food

**Comment by yclatious** [comment]
Merged fucking when.

**Comment by jknpj** [comment]
It didn't have to be over the top like this...

**Comment by RealestEstate** [comment]
I didn't see any harm in it.

**Pull request merged**